lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb

```diff
@@ -17,6 +17,7 @@
    "source": [
     "import numpy as np\n",
     "import matplotlib.pyplot as plt"
```

Comment:
```suggestion
    "import matplotlib.pyplot as plt",
```

Reply:
I suspect this is the reason the docs broke...

Reply:
that's right. How can I edit the from-dicom.ipynb in my local computer? using the Jupyter-lab? 
![image](https://user-images.githubusercontent.com/25382578/145152050-161287fe-e0e2-4bc9-846a-e043886648b8.png)

Reply:
Yeah, jupyterlab should do the trick :slightly_smiling_face: